experiment_pages/experiment/experiment_menu_ui.py

```diff
@@ -42,7 +42,7 @@ def __init__(self, parent: CTk, name: str, prev_page: ChangeableFrame = None, co
 
         self.collection_button = CTkButton(main_frame, text='Data Collection', width=button_size,
                                 command= self.data_page.raise_frame)
-        self.analysis_button = CTkButton(main_frame, text='Data Analysis', width=button_size,
+        self.analysis_button = CTkButton(main_frame, text='Data Exporting', width=button_size,

```

Comment:
Again, just a change of naming convention for UI. Let me know once the meat of the code is pushed and I'll report back.